packages/front-end/services/datasources.ts

```diff
@@ -77,7 +77,7 @@ WHERE
       ? ",\n  event_value_in_usd as value"
       : type === "binomial"
       ? ""
-      : `,\n  value_param.value.${type === "count" ? "int" : "float"}_value`
+      : `,\n  value_param.value.${type === "count" ? "int" : "float"}_value` as `value`
```

Comment:
Thanks for the contribution!  I think you have a couple extra backticks.  It should be the following instead:

```suggestion
      : `,\n  value_param.value.${type === "count" ? "int" : "float"}_value as value`
```

Reply:
Thanks for the pointer, but that's not correct.
In SQL you surround fields when you want to be explicit.
An example is when you have a field that's called "from", you need to surround it with backticks.
Otherwise the field name is taken as a keyword and you'll get an error running the query.

Reply:
This is inside a template string in Typescript, which also uses backticks, so the code you have now is a syntax error and will not compile since those internal backticks are not escaped.

Since we're hardcoding the column name to `value`, backtick escaping is not necessary like it would be if the column name was user-supplied.

Reply:
Ah, gotcha.

I just updated it and removed the extra backticks.
Thanks for pointing this out